test/Log/OpusLogTest.php

```diff
@@ -0,0 +1,211 @@
+<?php
+/**
+ * This file is part of OPUS. The software OPUS has been originally developed
+ * at the University of Stuttgart with funding from the German Research Net,
+ * the Federal Department of Higher Education and Research and the Ministry
+ * of Science, Research and the Arts of the State of Baden-Wuerttemberg.
+ *
+ * OPUS 4 is a complete rewrite of the original OPUS software and was developed
+ * by the Stuttgart University Library, the Library Service Center
+ * Baden-Wuerttemberg, the Cooperative Library Network Berlin-Brandenburg,
+ * the Saarland University and State Library, the Saxon State Library -
+ * Dresden State and University Library, the Bielefeld University Library and
+ * the University Library of Hamburg University of Technology with funding from
+ * the German Research Foundation and the European Regional Development Fund.
+ *
+ * LICENCE
+ * OPUS is free software; you can redistribute it and/or modify it under the
+ * terms of the GNU General Public License as published by the Free Software
+ * Foundation; either version 2 of the Licence, or any later version.
+ * OPUS is distributed in the hope that it will be useful, but WITHOUT ANY
+ * WARRANTY; without even the implied warranty of MERCHANTABILITY or FITNESS
+ * FOR A PARTICULAR PURPOSE. See the GNU General Public License for more
+ * details. You should have received a copy of the GNU General Public License
+ * along with OPUS; if not, write to the Free Software Foundation, Inc., 51
+ * Franklin Street, Fifth Floor, Boston, MA 02110-1301, USA.
+ *
+ * @category    Test
+ * @package     Opus
+ * @author      Kaustabh Barman <barman@zib.de>
+ * @copyright   Copyright (c) 2020, OPUS 4 development team
+ * @license     http://www.gnu.org/licenses/gpl.html General Public License
+ */
+
+namespace OpusTest\Log;
+
+use Opus\Log\LogService;
+
+class OpusLogTest extends \PHPUnit_Framework_TestCase
+{
+    private $opusLog;
+
+    private $tempFolder;
+
+    public function setUp()
+    {
+        parent::setUp();
+
+        $tempFolder = $this->createTempFolder();
```

Comment:
Do we actually need a temp folder for testing `Opus\Log`? The constructor takes a stream. Does it have to be a stream to a file?

Reply:
Will have to look at other alternatives if we don't want to have a temp folder. WIll go through the docs of ZendFramework once more. One problem I can think of now is how will we read the contents of the logger if it is not in a file?

Reply:
We had that topic before. When we create the stream object we can use a handle for something other than a file. That is PHP functionality.

Reply:
Done. Using PHP wrapper.